core/src/test/scala/io/finch/EndToEndSpec.scala

```diff
@@ -53,4 +55,15 @@ class EndToEndSpec extends FinchSpec {
     rep.contentString shouldBe "bar"
     rep.status shouldBe Status.Created
   }
+
+  it should "convert Endpoint[AsyncStream[Buf]] to service" in {
```

Comment:
I liked the previous version (a standalone test for `ToResponse`) more.

I think you can use `AsyncStream.fromReader` API to compare two async streams. Something like

```scala
val in = AsyncStream.fromSeq(...)

val out = implicitly[ToResponse.Aux[Plain.Text, Buf]](Ok(in))

assert(in == out)
```